checkov/terraform/graph_builder/local_graph.py

```diff
@@ -66,7 +67,8 @@ def __init__(self, module: Module) -> None:
         self.vertices_by_module_dependency: Dict[TFModule | None, Dict[str, List[int]]] = defaultdict(partial(defaultdict, list))
         self.enable_foreach_handling = strtobool(os.getenv('CHECKOV_ENABLE_FOREACH_HANDLING', 'True'))
         self.enable_modules_foreach_handling = strtobool(os.getenv('CHECKOV_ENABLE_MODULES_FOREACH_HANDLING', 'True'))
-        self.foreach_blocks: Dict[str, List[int]] = {BlockType.RESOURCE: [], BlockType.MODULE: []}
+        self.enable_datas_foreach_handling = strtobool(os.getenv('CHECKOV_ENABLE_DATAS_FOREACH_HANDLING', 'False'))
```

Comment:
We also have a file called `env_vars_config` with the full state of all env vars, could you also add it there and use it instead?
This gives us a unified place to handle all env vars, and also if we need to import them in multiple places we can do it easily :)

Reply:
done